src/Assignments/Monsters/students/11th/Kangyejiii/Monsters.js

```diff
@@ -30,15 +30,38 @@ class Monsters extends Component {
 
   // SearchBox 에 props로 넘겨줄 handleChange 메소드 정의
 
+  componentDidMount() {
+    fetch("https://jsonplaceholder.typicode.com/users")
+    .then(res=>res.json())
+    .then(res=>this.setState({monsters : res}));
+  }
+
+  handleChange = (e) => {
+    this.setState({userInput: e.target.value});
+    }
+
+    handleSearch = () => {
+      let result=this.state.monsters.filter((el)=>(el.name.toLowerCase().includes(this.state.userInput.toLowerCase())));
+      
+      if(!this.state.userInput.trim("")||result.length===0) {
```

Comment:
- 오오.. 정확한 비교를 위해 `trim()` 함수를 이용해 불필요한 space 를 제거해주는 부분 너무 인상깊네요! 👍 👍 👍 
- `result.length` 의 값이 `0` 인지 확인도 잘해주셨는데, `0` 은 falsy value 라서 `!result.length` 로 줄여줄 수도 있겠네요!

마찬가지로 코드스타일 조금만 지적하겠습니다 ㅎㅎㅎ
```suggestion
      if (!this.state.userInput.trim("") || result.length === 0) {
```